mail/api.py

```diff
@@ -131,7 +131,6 @@ def messages_for_recipients(recipients_and_contexts, template_name):
                 subject=subject,
                 body=text_body,
                 to=[recipient],
-                bcc=[settings.MAILGUN_BCC_TO_EMAIL] if settings.MAILGUN_BCC_TO_EMAIL else [],
```

Comment:
We use this in MM because when we send an email there, we use mailgun's API to send the email to multiple recipients but BCC only once. Here we're going to be generating *a lot* more emails and there's not much value of a BCC feature in OD.